src/settings/toml/manifest.rs

```diff
@@ -115,6 +120,35 @@ impl Manifest {
         Ok(template_config)
     }
 
+    pub fn worker_name(&self, env_arg: Option<&str>) -> String {
+        if let Some(environment) = self.get_environment(env_arg).unwrap_or_default() {
+            if let Some(name) = &environment.name {
+                return name.clone();
+            }
+            if let Some(env) = env_arg {
+                return format!("{}-{}", self.name, env);
```

Comment:
should be able to remove the `return` statements if you add an `else` clause to the `if let` that returns `self.name.clone()`

Reply:
i like the early return syntax?

Reply:
that's fair. not going to block on it. in this pattern though there are three paths to a return, and two of them use `return` and one does not. i'd be interested to see if clippy would complain about it - generally just feels inconsistent to me